site/content/en/docs/user/attach-configuration.md

```diff
@@ -0,0 +1,62 @@
+# Attach Configuration
+
+{{< hint "info" >}}
+
+This document walks you through how to configure the Attach feature.
+
+{{< /hint >}}
+
+## What is a Attach?
+
+The Attach API is a [`kwok` Configuration]({{< relref "/docs/user/configuration" >}}) that allows users to define and simulate attaching to Pod(s).
+
+A Attach resource has the following fields:
+
+``` yaml
+kind: Attach
+apiVersion: kwok.x-k8s.io/v1alpha1
+metadata:
+  name: <string>
+  namespace: <string>
+spec:
+  attaches:
+    - containers:
+        - <string>
+      logsFile: <string>
+```
+
+To attach a container, you can set the `attaches` field in the spec section of an Attach resource.
+The `containers` field is used to match an item in the `attaches` field. If the `containers` field is not set, the `attaches` item will default to all containers.
```

Comment:
The only thing I would add here is what does it mean to attach to all containers? Kubectl will default to the first one if no annotation is being set. Should we replicate that logic if no containers are specified?

Reply:
I got lost in notifications and am now updated.